m4_sessions/1901-inning/EI-matters.md

```diff
@@ -18,43 +18,37 @@ Students will learn a basic framework for understanding emotional intelligence a
 ## Session Outline
 
 
- 1. Introductions (10 minutes, lead by , see slide deck for intro format )
+ 1. Introductions (15 - 20 minutes, lead by , see slide deck for intro format )
+    - Icebreaker
  2. Intro to ei - Description and surface level explanation. (5 minutes, slides 3-6, lead by )
- 3. Article reading and discussion (20 minutes)
-     - We'll assign three groups. Each group gets 5 minutes to read one of the following articles:
-          1. [13 Signs of High Emotional Intelligence](https://www.inc.com/justin-bariso/13-things-emotionally-intelligent-people-do.html)
+ 3. Video and discussion (20 minutes)
+    - 3 - 5 minute video
+    - 15 minute discussion
+    - Share with room
+    - One person from each group: what stood out amongst what you   HEARD in your small group discussions?
 
-
-          2. [The Importance of Emotional Intelligence at Work](https://www.entrepreneur.com/article/245755)
-
-          3.[Inside Amazon: Wrestling Big Ideas in a Bruising Workplace](https://www.nytimes.com/2015/08/16/technology/inside-amazon-wrestling-big-ideas-in-a-bruising-workplace.html)
-     - Then, each group has a 10 minute discussion with the following prompts:
-
-          1.
-          * Have you paired with someone at Turing who you think demonstrates these traits? (13 traits article) Did that help your project? How?
-          * Which of these skills do you think you have? How has it helped you?
-          * Which of these skills do you wish you were better at? Why?
-
-          2.
-          * Have you worked with someone that exhibited these traits?
-          * Have you worked with someone that didn't exhibit these traits?
-
-          3.
-          * Do you think it's possible to be emotionally intelligent and do good work?
-          * Would you be willing to work in a place that's unwilling to acknowledge your feelings or mental state?
-
-      - Finally, each group shares a takeaway from their discussion with the larger group. (5 minutes)
-
-    4. Group activity, lead by Megan. (20 - 30 minutes)
+ 4. Group activity (20 - 30 minutes)
```

Comment:
What are you doing for this activity?

Reply:
The link on line 49 points to a flexible activity that we can choose from, based on time remaining at this point :)